microshift_backup_and_restore/microshift-auto-recover-manual-backup.adoc

```diff
@@ -6,25 +6,25 @@ include::_attributes/attributes-microshift.adoc[]
 
 toc::[]
 
-You can automatically restore data from manual backups when {microshift-short} fails to start by using the `auto-recovery` feature.
+You can automatically restore data from manual backups when {microshift-short} fails to start by using the `auto-recovery` feature. Use the following options with the existing `backup` and `restore` commands:
 
-You can use the following options with the existing `backup` and `restore` commands in this feature:
+* The `--auto-recovery` option selects the most recent version of the backup, and then restores it. This option treats the `PATH` argument as a path to a directory that holds all the backups for automated recovery, and not just as a path to a particular backup file.
```

Comment:
Maybe we could improve it more? `microshift restore --auto-recovery PATH` will select the most recent backup, `microshift backup --auto-recovery PATH` will create a new backup.
so `option selects the most recent version of the backup, and then restores it` for `microshift backup` sounds weird.

Maybe it should start with something like: changes the behavior of the command to perform procedures related to auto recovery, or something